### DISCUSSION TITLE
Can we keep the automation except when we use "+ Add Item"?

### DISCUSSION THREAD

**Discussion by hrobinkong** [leonsteinhaeuser/project-beta-automations#76] [General]
Hello! I'd open a feature request if unavailable, but I would like to address it here beforehand just in case I'm overlooking it.

Currently, our group has set up a board with several tabs, and we are using this workflow to automate issues to "Backlog".

![kanban](https://github.com/leonsteinhaeuser/project-beta-automations/assets/119588002/1c3fbd17-1363-4be1-98a8-9ad9e2f0e3d9)

We would like to keep this automation _unless_ we create an issue through the **"+ Add Item"** button in the bottom of each column. When we create an issue through the button in a specific column, we would like to keep the issues in the same column. (Currently, through the automation it moves the issues back to Backlog)

**Comment by leonsteinhaeuser** [comment]
Hello @hrobinkong.

this use case is currently not covered. The only thing that is possible is to remove the "issue open" trigger for the action. But this would also mean that **new issues** created directly in this repository would no longer be managed.
